changelog.md

```diff
@@ -1,4 +1,18 @@
 
+## 2.3.1 / 2019-07-05
+
+### Fixed issues
+
+- Search: #2001, #2016, #2025
+- Docker: #2005, #2054
+- Timeline: #2032
+- Sharedspace: #2058
```

Comment:
"shared space" (with a space between shared and space)

Reply:
Done